src/patternfly/components/ChipGroup/examples/ChipGroup.md

```diff
@@ -130,19 +130,51 @@ cssPrefix: pf-c-chip-group
 {{/chip-group-list-item}}
 {{/chip-group}}
 ```
-### Overiew 
+
+```hbs title=Toolbar-closable
+{{#> chip-group chip-group--modifier="pf-m-toolbar" chip-group--id="chip-group-toolbar-closable"}}
+  {{#> chip-group-list-item}}
+    {{#> chip-group-label chip-group-label--attribute=(concat 'id="' chip-group--id '-label"')}}
+      Category one
+    {{/chip-group-label}}
+    {{#> chip-group newcontext}}
+      {{#> chip}}
+        {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_one_toolbar"')}}
+          Chip one
+        {{/chip-text}}
+        {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_one_toolbar ' chip-group--id 'chip_one_toolbar" aria-label="Remove" id="' chip-group--id 'remove_chip_one_toolbar"')}}
+          <i class="fas fa-times-circle" aria-hidden="true"></i>
+        {{/button}}
+      {{/chip}}
+      {{#> chip}}
+        {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_two_toolbar"')}}
+          Chip two
+        {{/chip-text}}
+        {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_two_toolbar ' chip-group--id 'chip_two_toolbar" aria-label="Remove" id="' chip-group--id 'remove_chip_two_toolbar"')}}
+          <i class="fas fa-times-circle" aria-hidden="true"></i>
+        {{/button}}
+      {{/chip}}
+    {{/chip-group}}
+    {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id '-label remove_all_chips" aria-label="Remove all chips" id="remove_all_chips"')}}
```

Comment:
What do you think about creating a `chip-group__close-all` element or something like that that we style instead of styling the button component and it needing to be adjacent to the chip group? That would allow for flexibility if someone adds functionality to the button that creates a wrapper for some reason, future proofs for if we add another button that does something else, and we aren't styling a component in a component - which just gets tricky, though styling the margin currently doesn't pose much risk.